src/performance-analysis-spec/specific-hardware-and-software.md

```diff
@@ -85,8 +85,8 @@ GOT entries, including those used for the PLT, remain 64 bit.
 ## P128 (w/ data-dependency fix) (w/ larger store queue)
 
 This configuration is identical to the P128 Forced-GOT configuration except
-that access to globals is sometimes performed via PCC, bypas
-ing GOT entries in configurations where aarch64 would offer the same choice
+that access to globals is sometimes performed via PCC, bypassing GOT entries
```

Comment:
```suggestion
that access to globals is sometimes performed via PC, bypassing GOT entries
```
maybe? It’s PC-relative with an integer base so not using PCC but PC via DDC

Reply:
Oh, I see what you mean, in the P128 case. Yes.